src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -11214,5 +11214,119 @@ static unsafe void M(dynamic d, int* p)
                 Diagnostic(ErrorCode.ERR_BadBinaryOps, "d += p").WithArguments("+=", "dynamic", "int*").WithLocation(5, 9)
                 );
         }
+
+        [Fact, WorkItem(56646, "https://github.com/dotnet/roslyn/issues/56646")]
+        public void LiftedUnaryOperator_InvalidTypeArgument01()
```

Comment:
Consider testing a conversion operator whose parameter type is `TypedReference`

Reply:
How would you suggest getting a value to input into such a conversion?

Reply:
Oop, I think I see that this can't be done. Thanks.